Лабораторная работа 4

klimargarita opened the pull request.

klimargarita committed:
Лабораторная работа 4

AlxSy289 commented on Лабораторная работа 4/task_1.py:
```diff
@@ -0,0 +1,131 @@
+if __name__ == "__main__":
```
Это зачем?

AlxSy289 reviewed (changes requested):
Не хватает свойств и демонстрации создания и использования объектов

klimargarita committed:
Update task_1.py

Добавила свойства для работы с скрытыми данными. Показала создание и использование объектов

AlxSy289 reviewed (approved):
Хорошо
